gematria/datasets/convert_bhive_to_llvm_exegesis_input.cc

```diff
@@ -29,8 +29,9 @@
 #include "gematria/llvm/llvm_architecture_support.h"
 #include "gematria/utils/string.h"
 
-constexpr uint64_t kInitialRegVal = 0x10000;
-constexpr uint64_t kInitialMemVal = 0x7FFFFFFF;
+constexpr uint64_t kInitialRegVal = 0x12345600;
```

Comment:
add a comment about where the numbers came from, because they are quite odd otherwise

do we understand why they are better, exactly?

Reply:
Comment added along with an explanation for the second question. Essentially they are high enough to avoid accesses in the first page/underflow, but low enough where most accesses are below the virtual address space ceiling. They're not perfect, but anecdotally they work reasonably well.